src/hooks/screen/useChatInfoHook.ts

```diff
@@ -24,15 +33,10 @@ function useChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
     selectedUser,
     blockModalRef,
     isLoadingInitChat
-  } = useGroupInfo();
-  const {localDb} = useLocalDatabaseHook();
-  const [loadingChannelInfo, setLoadingChannelInfo] = React.useState<boolean>(false);
-  const {isLoadingFetchingChannelDetail, selectedChannel, goBack} = useChatUtilsHook();
-  const [myUserId] = React.useContext(Context).profile;
-  const navigation = useNavigation();
-  const [showPopupBlock, setShowPopupBlock] = React.useState(false);
-  const [channelInfo, setChannelInfo] = React.useState(null);
+  } = useGroupInfo(selectedChannel?.id);
+
   const {signedProfileId, anonProfileId} = useUserAuthHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useGroupInfo` hook is now being called with the `selectedChannel?.id` parameter. This change could potentially introduce a bug if `selectedChannel` is null or undefined, as it would result in calling `useGroupInfo` with an undefined parameter. Please ensure that there are appropriate null checks or default values in place within the `useGroupInfo` hook to handle this scenario.
```diff
-   } = useGroupInfo(selectedChannel?.id);
+   } = selectedChannel ? useGroupInfo(selectedChannel.id) : useGroupInfo();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->